content/1.7/back-office/shop-parameters/search/search-listing.md

```diff
@@ -0,0 +1,108 @@
+
+- Select all/Unselect all: this button will select/unselect all the checkboxes at the beginning of each row of the listing.
+- Enable selection. Enables the selected alias, changes the status to green check mark. If no alias is selected when pressing "enable selection" the message is shown: "You must select at least one item to perform a bulk action." If an alias is selected, and "enable" is pressed, the message is shown: "The status has been successfully updated." and check mark is changed to a green one.
+- Disable selection. Disables the selected alias, changes the status to red cross mark. If no alias is selected when pressing "disable selection" the message is shown: "You must select at least one item to perform a bulk action." If an alias is selected, and "disable selection" is pressed, the message is shown: "The status has been successfully updated." and check mark is changed to a red cross.
+- Delete selected: This button will delete the selected alias. 
+After clicking on "Delete selected", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" (See issue [#14462](https://github.com/PrestaShop/PrestaShop/issues/14462)). The action can be canceled by clicking on the cross or on the cancel button.
+After clicking on "Delete" button, the selected feature values are deleted, a successful message is displayed "The selection has been successfully deleted."
+
+ ## Search
+ 
+The search button is disabled as long as there are less than 2 alias  in the list. A search can be done by aliases, search, and status.
+It's possible to sort the list by aliases, search in an ascending or descending way.
+After clicking on "search", the "search" button is disabled and a "reset" button is displayed.
+If there is no results for the search done, the following message is displayed: "No records found".
+When clicking on "reset", the search is removed and all the results are displayed.
+
+ ## Indexing
+ 
+This section provides information on the number of products that can be searched through shop's search function, and compares it to the number of products present in the catalog.
+
+Firstly an explanation is shown: "The "indexed" products have been analyzed by PrestaShop and will appear in the results of a front office search.
+Indexed products 18 / 18." (the number depends on the products which are in the shop, and are indexed).
+Below folows: "Building the product index may take a few minutes. If your server stops before the process ends, you can resume the indexing by clicking "Add missing products to the index"."
+Below there are 2 links that when pressed performs the following actions:
+ -Add missing products to the index - only the new products will be indexed. Also when pressed, the page will be refreshed, and a message shown: "Successful update."
+ -Re-build the entire index - the "Re-build entire index" process takes more time, but it is more thorough.
```

Comment:
```suggestion
 -"Re-build the entire index" : the "Re-build entire index" process takes more time, but it is more thorough.
```

Reply:
It's not really helpful to know that it takes more time, furthermore, this can depend on a lot of things (server, connection ....). I think you can remove this part and rather explain what happens when clicking on it: all the products are indexed and not only the missing ones compared to the previous link